packages/component-library/components/Icon/Icon.module.scss

```diff
@@ -4,10 +4,6 @@
 .icon {
   @extend %caIcon;
 
-  // Set sensible defaults for icon colours when
-  // windows high-contrast mode is enabled. These
-  // should be overridden by the consumer when the
-  // icon colour conveys information, such as mood.
```

Comment:
Oh yeah, @HeartSquared these comments were only picked up in Icons and one other component I think when adding all the components. So shouldn't be too disruptive later